### PR TITLE
Normalize makefiles for windows compatibility

### PR DESCRIPTION
**Pull request by wgsuse**
This PR normalizes the makefiles  so they are compatible with windows.

### CODE REVIEW

File: cmd/gen-release-notes/Makefile [hunk 1]
```diff
@@ -1,20 +1,25 @@
-GO = go
+.PHONY: clean
```
Comment (briandowns) [inline cmd/gen-release-notes/Makefile:1]: This should be placed above the target it corresponds to.

File: cmd/standup/Makefile [hunk 2]
```diff
@@ -11,7 +18,7 @@ UNAME_S = $(shell uname -s)
 MACOS_MANPAGE_LOC = /usr/share/man
 LINUX_MAPPAGE_LOC = $(PREFIX)/man/man8
 
-$(BINDIR)/$(BINARY): $(BINDIR) clean
+$(BINDIR)/$(BINARY): $(BINDIR) main.c
```
Comment (briandowns) [inline cmd/standup/Makefile:21]: Was this change tested?

File: cmd/k3s-release/Makefile [hunk 3]
```diff
@@ -1,16 +1,25 @@
-GO = go
+.PHONY: clean
```
Comment (briandowns) [inline cmd/k3s-release/Makefile:1]: This should be placed above the corresponding target.

File: cmd/k3s-release/Makefile [hunk 4]
```diff
@@ -1,16 +1,25 @@
...
+BINARY := k3s-release$(EXT)
 
+GO      = go
 VERSION = v0.1.0
 GIT_SHA = $(shell git rev-parse HEAD)
 LDFLAGS = -ldflags "-X main.gitSHA=$(GIT_SHA) -X main.version=$(VERSION) -X main.name=$(BINARY)"
 TAGS    = "netgo osusergo no_stage static_build"
-$(BINDIR)/$(BINARY): clean
+
+$(BINDIR)/$(BINARY): $(BINDIR) $(shell ls *.go)
```
Comment (briandowns) [inline cmd/k3s-release/Makefile:17]: Why are we listing out all of the go files there?

**Comment by wgsuse** [comment]
I'm closing this one. The source tree and Makefiles have changed a lot. I'll revisit this again and submit a new PR

**Pull request closed without merging**